cdc/owner.go

```diff
@@ -296,11 +296,11 @@ func (o *Owner) newChangeFeed(
 	processorsInfos model.ProcessorsInfos,
 	taskPositions map[string]*model.TaskPosition,
 	info *model.ChangeFeedInfo,
-	checkpointTs uint64) (cf *changeFeed, resultErr error) {
```

Comment:
But if the function exits with `return nil, err`, the `resultErr` will be assigned to value `err`

Reply:
got it, it would be better to make it explicit ?

Reply:
Yes, explicit is better
```